README.md

```diff
@@ -8,50 +8,10 @@ This project uses the GNOME technology stack, including GTK and WebKit for UI, a
 
 Building
 --------
-We recommend using [JHbuild] to build this code, especially if you are planning to make changes to it.
-A sample JHbuild moduleset is included in the root directory of this repository.
+We recommend using [Flapjack] to build this code, especially if you are planning to make changes to it.
+The [example.flapjack.ini] configuration file included with Flapjack is precisely made to build the `eos-knowledge-lib` and other components,
```

Comment:
There's a comma at the end of this sentence — did you intend to write more after it?

Reply:
Good catch.

Reply:
Good catch.